### ISSUE TITLE
Do not expose class names/IDs as vars

### ISSUE THREAD

**Issue by roman01la** [mhallin/forest#5]
Hi! Thanks for this cool library. I wonder why you decided to expose class names and IDs as vars. Wouldn't it be less confusing to make a stylesheet var to be a map?

**Comment by mhallin** [comment]
I agree it's a bit magic that the class names become vars, but the purpose is to get as much help from the compiler as possible when maintaining the stylesheets: you get warnings when you reference classes that don't exist, or remove classes that are used by other parts of the code.
